locales-pending/premium.ftl

```diff
@@ -298,6 +298,22 @@ data-broker-profiles-estimated-time = Est. time to complete: { $estimated_time }
 # $exposure_reduction is the percentage of exposures that are data brokers.
 data-broker-profiles-exposure-reduction = Exposure reduction: { $exposure_reduction }%
 
+# Welcome to Premium Data Broker Profiles
+
+welcome-to-premium-data-broker-profiles-title-part-one = Welcome to Premium.
+welcome-to-premium-data-broker-profiles-title-part-two = We’ll remove those profiles ASAP.
```

Comment:
Q: Can't remember where we discussed earlier, but I think @flodolo was saying phrasing like "ASAP" might not translate well in other locales. Not sure if we have a ticket w/ UX/content folks to possibly tweak this.

Reply:
We should explain it in a comment, but I think it doesn't look great even for English.

Reply:
Started the discussion w prod/design